AsyncDisplayKit/ASMapNode.mm

```diff
@@ -161,7 +161,18 @@ - (MKCoordinateRegion)region
 
 - (void)setRegion:(MKCoordinateRegion)region
 {
-  self.options.region = region;
+  MKMapSnapshotOptions * __weak oldOptions = self.options;
+  MKMapSnapshotOptions * options = [[MKMapSnapshotOptions alloc] init];
+  options.camera = oldOptions.camera;
+  options.mapRect = oldOptions.mapRect;
+  options.mapType = oldOptions.mapType;
+  options.showsPointsOfInterest = oldOptions.showsPointsOfInterest;
+  options.showsBuildings = oldOptions.showsBuildings;
+  options.size = oldOptions.size;
+  options.scale = oldOptions.scale;
+  options.region = region;
+  self.options = options;
+//  self.options.region = region;
```

Comment:
@appleguy @Adlai-Holler I had written it with `[self.options copy]` before, but it didn't work for some reason, that's why I did what I did there. But it seems I must have done something wrong somewhere else because it seems to work fine with `[self.options copy]` now.

Thanks!